frontend/packages/kubevirt-plugin/src/selectors/config-map/sc-defaults.ts

```diff
@@ -19,24 +19,29 @@ const getSCConfigMapAttribute = (
   );
 };
 
-export const getDefaultSCAccessMode = (
+export const getDefaultSCVolumeMode = (
   storageClassConfigMap: ConfigMapKind,
   storageClassName: string,
 ) =>
   getSCConfigMapAttribute(
     storageClassConfigMap,
     storageClassName,
-    'accessMode',
-    PVC_ACCESSMODE_DEFAULT,
+    'volumeMode',
+    PVC_VOLUMEMODE_DEFAULT,
   );
 
-export const getDefaultSCVolumeMode = (
-  storageClassConfigMap: ConfigMapKind,
-  storageClassName: string,
-) =>
+const getDefaultSCAccessMode = (storageClassConfigMap: ConfigMapKind, storageClassName: string) =>
   getSCConfigMapAttribute(
     storageClassConfigMap,
     storageClassName,
-    'volumeMode',
-    PVC_VOLUMEMODE_DEFAULT,
+    'accessMode',
+    PVC_ACCESSMODE_DEFAULT,
   );
+
+export const getDefaultSCAccessModes = (
+  storageClassConfigMap: ConfigMapKind,
+  storageClassName: string,
+) => {
+  const defaultMode = getDefaultSCAccessMode(storageClassConfigMap, storageClassName);
+  return defaultMode ? [defaultMode] : [];
```

Comment:
Considering implementation of the `getDefaultSCAccessMode`, this condition is not needed.
I would even merge these two `getDefaultSCAccessMode() and getDefaultSCAccessModes()` into a single function.

Reply:
you are right, fixed